packages/astro/src/vite-plugin-build-html/index.ts

```diff
@@ -140,7 +137,7 @@ export function rollupPluginAstroBuildHTML(options: PluginOptions): VitePlugin {
           for (let node of findAssets(document)) {
             if (isBuildableLink(node, srcRoot)) {
               const href = getAttribute(node, 'href')!;
-              const linkId = viteifyPath(href);
+              const linkId = `/@fs${href}`;
```

Comment:
I didn’t want to affect the build with the “viteify“ change, so I manually left this as it was. cc @matthewp what do you think?